tcp/tcp.py

```diff
@@ -98,12 +108,23 @@ def __process_requests(self, client_socket, client_addr):
             date = datetime.now()
             dt_string = date.strftime('%Y%m%d%H%M.%S')
             dt_string = 'busybox date -s ' + dt_string + "\n" + "EOM\n"
+            req_dict = {   # this dict is used only when the received command results in logging only
+                RECV_RECEIVED_DATE: lambda: logger.info("TCP - pi successfully received date string"),
+                RECV_SET_DATE: lambda: logger.info("TCP - pi successfully set local datetime"),
+                RECVD_MOUNTING: lambda: logger.info("TCP - pi mounting has started"),
+                RECVD_MOUNTED: lambda: logger.info("TCP - pi successfully mounted filesystem"),
+                RECVD_UNMOUNTING: lambda: logger.info("TCP - pi unmounting has started"),
+                RECVD_UNMOUNTED: lambda: logger.info("TCP - pi successfully unmounted filesystem"),
+                RECVD_BOOT: lambda: logger.info("TCP - pi successfully received boot command"),
+                RECVD_FORMAT: lambda: logger.info("TCP - pi successfully received format command"),
+                RECVD_REINSTALL: lambda: logger.info("TCP - pi successfully received reinstall command")
+            }
             print("Sending date command to pi:", dt_string)
             client_socket.send(dt_string.encode())
             req = fd.readline()
             while req:
                 req = req.strip()
-                logger.info("TCP - recieved request {}".format(req))
+                logger.info("TCP - received request {}".format(req))
                 if req == RECV_IS_UNINSTALLED:
```

Comment:
it would be nice to perhaps include these in the dictionary to be consistent.

Reply:
I could include these in the dictionary, but I would have to create a separate function for the if statements with multiple statements since lambda expressions only allow a single statement. I left them as is because if they were moved to separate functions the code would be in a different place from the rest of the related logic. Should I move those to functions?

Reply:
it's totally up to you.

Reply:
I think it is more clear as is. If they were moved to separate functions, client_sock would have to either be passed as parameter (to only be used in two of the functions/lambdas) or added as an instance variable and I would need to add something to handle the break statement.